modules/core/src/matrix_transform.cpp

```diff
@@ -368,152 +368,177 @@ template<typename T1, typename T2> CV_ALWAYS_INLINE void flipHoriz_double( const
 }
 #endif
 
-static void
-flipHoriz( const uchar* src, size_t sstep, uchar* dst, size_t dstep, Size size, size_t esz )
+class flipHorizComputer : public ParallelLoopBody
 {
+public:
+    flipHorizComputer(const uchar* src0, size_t sstep, uchar* dst0, size_t dstep, Size size, size_t esz)
+        : src0_(src0), sstep_(sstep), dst0_(dst0), dstep_(dstep), size_(size), esz_(esz) {
+#if !CV_SIMD
+        limit_ = (int)(((size.width + 1)/2)*esz);
```

Comment:
Please fix this compilation error (GCC 7.5.0):
```
/opencv/modules/core/src/matrix_transform.cpp:374:5: error: uninitialized const member in ‘const int’ [-fpermissive]
     flipHorizComputer(const uchar* src0, size_t sstep, uchar* dst0, size_t dstep, Size size, size_t esz)
     ^~~~~~~~~~~~~~~~~
/opencv/modules/core/src/matrix_transform.cpp:535:15: note: ‘const int cv::flipHorizComputer::limit_’ should be initialized
     const int limit_;
               ^~~~~~
```

Reply:
Sorry for the typo.
Before commit I add const to limit_ and intuitively think it is O.K. However it actually makes compile fail.
Sorry for the inconvenience.